README.md

```diff
@@ -1,57 +1,18 @@
-# Northcoders House of Games API
+# NC Backend Project
 
-## Background
+## Project Trello
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+https://trello.com/b/zlUxNoND/be-nc-games
 
-Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
+## Set Up Instructions
 
-## Kanban
+1. Environment Variables.
```

Comment:
The user will need to know where to make these files, what to put inside them, specifically which file needs to reference which specific database. We can't assume they'll know any of this